contrib/TeXShop/auto-configure.command

```diff
@@ -61,13 +61,25 @@ defaults write "$TeXShopDir/Contents/Info.plist" CFBundleDocumentTypes -array-ad
 
 echo "Adding Gregorio file extensions to appropriate preference lists"
 #enable syntax coloring and the Typeset button for gabc files
-defaults write TeXShop OtherTeXExtensions -array-add "gabc"
-defaults write TeXShop OtherTeXExtensions -array-add "gtex"
+TeXExtensions=$(defaults read TeXShop OtherTeXExtensions 2>/dev/null)
+if [[ ! $TeXExtensions == *"gabc"* ]]; then
```

Comment:
Mostly because the [model](http://stackoverflow.com/questions/229551/string-contains-in-bash) I was looking at for this sort of test did so as well.  I still don't understand 100% when quoting is necessary in a bash script and when it is not.